react/modules/gaUpdateEcommerce.ts

```diff
@@ -0,0 +1,13 @@
+import push from './push'
+
+export default function updateGA4Ecommerce(
+  event: string,
+  data: Record<string, unknown>
+) {
+  push({ ecommerce: null })
```

Comment:
I took from [this conversation](https://vtex.slack.com/archives/C03KUDM8HSS/p1676381635313649?thread_ts=1675954481.118009&cid=C03KUDM8HSS) that we should also push this.

```suggestion
  push({ ecommerce: null })
  push({ ecommerceV2: null })  
```